content/docs/kitty/building.md

```diff
@@ -108,20 +111,20 @@ directory to your `PATH`.
 ```sh
 export PATH=$(pwd)/arm-gnu-toolchain-12.3.rel1-aarch64-none-elf/bin:$PATH
 ```
+
 ## Compiling the Kitty system
 
 The Kitty system, when running, takes files from an NFSv3 server.  The
 address of this server has to be known at build time.
 
 ```sh
-cd examples/kitty
 # IP adddress of NFS server to connect to
 export NFS_SERVER=0.0.0.0
 # NFS directory to mount
 export NFS_DIRECTORY=/path/to/dir
 export MICROKIT_SDK=/path/to/sdk
-# Compile the system
-make
+# Initialise submodules and compile the system
+make submodules all
```

Comment:
They should have already initialised the submodules by now, so all that's needed is `make`.